### ISSUE TITLE
[Question] lora finetune loss=0.00

### ISSUE THREAD

**Issue by Sarahtorekryo** [haotian-liu/LLaVA#816]
### Question

Hi Haotian~ I got training loss=0.00 when doing lora finetuning on custom data. I've checked that:

**Comment by aneet-javis** [comment]
@Sarahtorekryo I'm also trying to fine-tune LLaVA on custom dataset. Could you share dataset format/resources or code to start with? Thanks.

**Comment by haotian-liu** [comment]
model_max_length means how many tokens are in total, this includes everything in the conversation, and image tokens (576).

Any content that is larger than model_max_length will be truncated.

**Comment by Sprinter1999** [comment]
Hi,I encountered the same issue. I input more than 2 images for SFT. Have you managed to resolve it? And I wonder where to set model_max_length?

**Comment by Sprinter1999** [comment]
> Hi,I encountered the same issue. I input more than 2 images for SFT. Have you managed to resolve it? And I wonder where to set model_max_length?

Oh I've figured it out, thank you guys.

**Comment by lqqyyy** [comment]
same issue.
